src/main/java/duke/command/DeadlineCommand.java

```diff
@@ -0,0 +1,37 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.task.Deadline;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+import java.time.LocalDateTime;
+
+/**
+ * Represents a deadline command. Upon execution, updates data/duke.txt and produces some feedback to the user.
+ */
+public class DeadlineCommand extends Command {
```

Comment:
minor gripe. you could make this a final class to show that there it is non extensible.

Reply:
Thanks for your suggestion, but i think this class is possibly extended by other class in future. For example, I might create the subcategory of deadline class in the future which extend `Deadline` class.